third_party/amd/include/TritonAMDGPUToLLVM/TargetUtils.h

```diff
@@ -0,0 +1,24 @@
+#ifndef TRITON_CONVERSION_TRITONGPU_TO_LLVM_TARGETUTILS_H
+#define TRITON_CONVERSION_TRITONGPU_TO_LLVM_TARGETUTILS_H
+
+#include "llvm/ADT/StringRef.h"
+
+namespace mlir::triton::AMD {
+
+// A list of ISA families we care about.
```

Comment:
Just want to clarify, do we care about old archs, like CDNA1 and RDNA1?

Reply:
Probably not seriously, in the short term. But if we can have them covered along the way that's good too.